frontend-svelte/src/components/widgets/inputs/MenuBarInput.svelte

```diff
@@ -76,11 +74,10 @@
 </script>
 
 <div class="menu-bar-input" bind:this={self} data-menu-bar-input>
-	{#each entries as entry, index (index)}
+	{#each entries as entry, index}
```

Comment:
So I can learn Svelte better, is this redundant or is there some other reason not to do this? I'm used to keys being mandatory in Vue.

Reply:
There is no point of using a key in svelte. There is no vdom, so the whole item would get replaced.

Use (keyed) only if the item inside #each won't change, but only the props to the components.

There is no point of using array index as key at all.

In svelte, only use keyed each loop if performance is really bad.

Reply:
I see, thanks for the explanation. I'll clean up all the cases where we use an index then over the coming days.

Reply:
On this highlighted line, you can remove the `, index` part also since it's not used within the HTML, right?

Reply:
yes

Reply:
You don't need to clean up other cases. It's a waste of time with no clear benefit.